rules/game_process.tex

```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands, covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}, straight over its head.
+The GameController signal for the transition from \texttt{initial} to \texttt{ready} will be delayed by \qty{\SetDelayTimeChampion}{\second} for champions cup and by \qty{\SetDelayTimeChallenge}{\second} for challenge shield. 
```

Comment:
Minor edits: "for champions cup" -> "for the Champions Cup" and "for challenge shield" -> "for the Challenge Shield". 
I recommend searching for other occurrences - as these are proper nouns, they should be capitalized each time.